specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2022-10-01/ResourceHealth.json

```diff
@@ -0,0 +1,2322 @@
+            },
+            "occuredTime": {
+              "type": "string",
+              "description": "Timestamp for when last change in health status occurred.",
+              "format": "date-time"
+            },
+            "reasonChronicity": {
+              "type": "string",
+              "description": "Chronicity of the availability transition.",
+              "enum": [
+                "Transient",
+                "Persistent"
+              ],
+              "x-ms-enum": {
+                "name": "reasonChronicityTypes",
+                "modelAsString": true
+              }
+            },
+            "reportedTime": {
+              "type": "string",
+              "description": "Timestamp for when the health was last checked. ",
+              "format": "date-time"
+            },
+            "recentlyResolved": {
+              "type": "object",
+              "description": "An annotation describing a change in the availabilityState to Available from Unavailable with a reasonType of type Unplanned",
+              "properties": {
+                "unavailableOccuredTime": {
+                  "type": "string",
+                  "description": "Timestamp for when the availabilityState changed to Unavailable",
+                  "format": "date-time"
+                },
+                "resolvedTime": {
+                  "type": "string",
+                  "description": "Timestamp when the availabilityState changes to Available.",
+                  "format": "date-time"
+                },
+                "unavailableSummary": {
+                  "type": "string",
+                  "description": "Brief description of cause of the resource becoming unavailable."
+                }
+              }
+            },
+            "recommendedActions": {
+              "type": "array",
+              "description": "Lists actions the user can take based on the current availabilityState of the resource.",
+              "items": {
+                "$ref": "#/definitions/recommendedAction"
+              },
+              "x-ms-identifiers": [
+                "action"
+              ]
+            },
+            "serviceImpactingEvents": {
+              "type": "array",
+              "description": "Lists the service impacting events that may be affecting the health of the resource.",
+              "items": {
+                "$ref": "#/definitions/serviceImpactingEvent"
+              },
+              "x-ms-identifiers": [
+                "correlationId"
+              ]
+            }
+          }
+        }
+      }
+    },
+    "operationListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/operation"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "List of operations available in the resourcehealth resource provider."
```

Comment:
>resourcehealth [](http://example.com/codeflow?start=62&length=14)

nit, should it use the actual provider name 'Microsoft.ResourceHealth'? Or go for fully conversational style? 'the resource health resource provider'?

Reply:
fixed